.github/workflows/release.yml

```diff
@@ -41,6 +41,25 @@ jobs:
           python setup.py sdist bdist_wheel
           twine upload dist/*
 
+  publish-python-sdk-docker:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+    needs:
+      - publish-python-sdk
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build Merlin SDK Docker py${{ matrix.python-version }}
+        working-directory: ./python/sdk
+        run: |
+          IMAGE_TAG="ghcr.io/caraml-dev/merlin-sdk:py${{ matrix.python-version }}-${{ inputs.version }}"
+          SDK_VERSION="$(sed 's/^v\(.*\)-/\1/' <<< "${{ inputs.version }}")"
+          docker login ${{ env.DOCKER_REGISTRY }} -u ${{ github.actor }} -p ${{ secrets.ghcr_token }}
```

Comment:
Why not use the `docker/login-action@v1` action for this like some of the other publish jobs?